src/python/turicreate/toolkits/image_classifier/_annotate.py

```diff
@@ -72,6 +72,10 @@ def annotate(data, image_column=None, annotation_column='annotations'):
         [4 rows x 3 columns]
 
     """
+    # Check if Value is Empty
+    if __tc.SFrame.num_rows(data)== 0:
```

Comment:
If that `data` is not even an SFrame, then when this line is evaluated the user will get an error message which is very difficult to understand. Let's move this code to line 79, after we've checked that `data` is an SFrame.

Calling `__tc.SFrame.num_rows(data)` to get the number of row will work but doing either `len(data)` or `data.num_rows()` is more standard and probably better.

One very minor note - the standard Python style guide lines (i.e. [PEP 8](https://pep8.org)) says you should have a space on both sides of a `==`.